nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -331,22 +331,26 @@ def test_forward_sigmoid():
     _test_sigmoid(np.random.uniform(size=(3, 4, 4, 3)).astype('float32'))
 
 #######################################################################
-# Argmin/Argmax
-# -------------
-
-def _test_argx(func, data, **kwargs):
+# Reduce
+# ------
 
+def _test_reduce(func, data, output_type=None, **kwargs):
     with tf.Graph().as_default():
         inp = array_ops.placeholder(shape=data.shape, dtype=data.dtype, name="c0")
-        func(inp, name="argx0", **kwargs, output_type=tf.int32)
+        if output_type is None:
+            func(inp, name="reducex0", **kwargs)
+        else:
+            func(inp, name="reducex0", output_type=output_type, **kwargs)
+
 
-        compare_tf_with_tvm(data, 'c0:0', 'argx0:0')
+        compare_tf_with_tvm(data, 'c0:0', 'reducex0:0')
 
-def test_forward_argminmax():
+def test_forward_reduce():
```

Comment:
OK, I'll revert it and newly add the reduce test.
Since I thought that argmin and argmax are with other reduce operators in tvm (c.f. [reduction.py](https://github.com/dmlc/tvm/blob/master/nnvm/python/nnvm/top/reduction.py)), I unified the test as `reduce` operator. If it is not intuitive, there is no problem even divided the test. Thanks.